<ui version="4.0" >
 <class>Dialog_vano_linkerloader</class>
 <widget class="QDialog" name="Dialog_vano_linkerloader" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>772</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>772</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Linker file editor</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>20</y>
     <width>731</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_rawimg" >
        <property name="text" >
         <string>Raw Image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineEdit_rawimg" />
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="pushButton_rawimg" >
        <property name="text" >
         <string>File ...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_maskimg" >
        <property name="text" >
         <string>Mask Image</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lineEdit_maskimg" />
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="pushButton_maskimg" >
        <property name="text" >
         <string>File ...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_apofile" >
        <property name="text" >
         <string>Annotation file</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="lineEdit_apofile" />
      </item>
      <item row="2" column="2" >
       <widget class="QPushButton" name="pushButton_apofile" >
        <property name="text" >
         <string>File ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Tip: if any texts above become red color, the files do NOT exist!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_loaddata" >
        <property name="text" >
         <string>Load data</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
